tests/test_memory.py

```diff
@@ -97,28 +97,6 @@ def render_foo(self, data):
         assert [r['foo'] for r in table.rows] == [-1, -2]
 
 
-def test_caches():
```

Comment:
Is this not a feature we depend on? Did you investigate the source of the intermittency?